packages/core/lib/commands/packages.js

```diff
@@ -0,0 +1,24 @@
+var command = {
+  command: "packages",
+  description: "List available packages on connected EthPM Registry",
+  builder: {},
+  help: {
+    usage: "truffle packages",
+    options: []
+  },
+  run: async function (options, done) {
+    var Config = require("@truffle/config");
+    var PackageV3 = require("@truffle/ethpm-v3");
+
+    var config = Config.detect(options);
+
+    if (config.ethpm.version == "1" || config.ethpm.version == "3") {
```

Comment:
Does PackageV3 support listing packages for v1?

Reply:
Yeah good catch, it doesn't, but I've gone ahead and updated `ethpm-v1` to mimic the behavior in `ethpm-v3` to display info about the release on the v1 ethpm registry.